utils/not_doctested.txt

```diff
@@ -710,6 +710,7 @@ src/transformers/models/pegasus_x/modeling_pegasus_x.py
 src/transformers/models/perceiver/configuration_perceiver.py
 src/transformers/models/perceiver/convert_perceiver_haiku_to_pytorch.py
 src/transformers/models/persimmon/convert_persimmon_weights_to_hf.py
+src/transformers/models/persimmon/modeling_persimmon.py
```

Comment:
```suggestion
src/transformers/models/persimmon/modeling_persimmon.py # OOM 
```

Reply:
I don't think so far we do any parsing the line (of this file) to get the actual path. The whole line is used - and that's why I mentioned we should probably need another file (or format, or extra parsing) in order to recode the information.

Reply:
I see! Ok let’s go with this for now!